pkg/scalers/cron_scaler.go

```diff
@@ -169,7 +169,7 @@ func (s *cronScaler) GetMetrics(ctx context.Context, metricName string, metricSe
 
 	/*******************************************************************************/
 	metric := external_metrics.ExternalMetricValue{
-		MetricName: cronMetricName,
```

Comment:
`cronMetricName` definition should be removed as well

Reply:
Removed!